pkg/util/iptables/iptables.go

```diff
@@ -348,7 +348,7 @@ func (runner *runner) SaveInto(table Table, buffer *bytes.Buffer) error {
 	// run and return
 	iptablesSaveCmd := iptablesSaveCommand(runner.protocol)
 	args := []string{"-t", string(table)}
-	klog.V(4).Infof("running %s %v", iptablesSaveCmd, args)
+	klog.V(4).InfoS("running iptables command", "iptablesSaveCmd", iptablesSaveCmd, "args", args)
```

Comment:
```suggestion
	klog.V(4).InfoS("Running iptables command", "command", iptablesSaveCmd, "args", args)
```

Reply:
The migration rules say to not use abbrevations for key names, so that should be `"arguments"` not `"args"` right? Or if "args" is one of the acceptable exceptions to the abbrevation rule it should probably be added to the doc.

(I'd be happier if we just had `"command"` here with the whole command (`iptablesSaveCmd` plus `args`) but that would require constructing that array just so we can log it...)